js/core/stdio/stdout.js

```diff
@@ -0,0 +1,31 @@
+// Copyright 2014-2015 runtime.js project authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+class stdout {
+  constructor() {
+
+  }
+
+  write(text, fg, bg) {
+    runtime.tty.print(text, 1, fg, bg);
+  }
+
+  writeln(text, fg, bg) {
```

Comment:
`defaultStdout` could be exposed as `runtime.tty.stdout` (different from class `runtime.tty.Stdout`)